Dockerfile

```diff
@@ -30,6 +30,13 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 # See https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58
 FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
 COPY --from=builder /opt/app-root/src/manager /
+
+# It is mandatory to set these labels
```

Comment:
To be a little cleaner I think you can set multiple LABELs at once like
```
LABEL \
    description=
    io.k8s.description=
```
https://dockerlabs.collabnix.com/beginners/dockerfile/Label_instruction.html

Reply:
updated